src/routes/api/api-router.ts

```diff
@@ -142,6 +143,7 @@ ApiRouter.post("/recover-client-key", RecoverClientKeyPost);
 ApiRouter.post("/re-encrypt-ghes-app", ReEncryptGitHubServerAppKeysPost);
 ApiRouter.use("/data-cleanup", DataCleanupRouter);
 ApiRouter.post("/recover-commits-from-date", RecoverCommitsFromDatePost);
+ApiRouter.post("/reset-failed-pending-deployment-cursor", ResetFailedAndPendingDeploymentCursorPost);
 
```

Comment:
What is the use case of this admin API? Shouldn't we create a generic API to reset cursor for build, deployment, commit and other tasks?

Reply:
Nah, I'd prefer a specific api for this PR. 
I want to remove the api once done. 
Do not want more api along with the existing TWO restart-sync api admin endpoints.